cli/init.ts

```diff
@@ -30,6 +39,9 @@ export async function handleInit() {
     .join(' ');
   spawnProcess(`npm install --save-dev ${devDependencyPackages}`);
 
+  // https://staging.coda.io/d/Quality-Tracker_d-GJF-DmEUK/Bugs_sucBW#Bugs_tuq-W/r20768&modal=true
```

Comment:
Are you sure you want a link to staging in a public repo?

Reply:
ah, good catch. totally forgot this is a public repo..